app/assets/javascripts/discourse/app/templates/components/bootstrap-mode-notice.hbs

```diff
@@ -4,7 +4,7 @@
       {{this.message}}
     </div>
     <div class="col-button">
-      <div class="alert--button"><DButton @icon="user-plus" @href="/my/invited" @class="btn-primary bootstrap-invite-button" @label="bootstrap_invite_button_title" /></div>
+      <div class="alert--button"><DButton @icon="user-plus" @action="inviteUsers" @class="btn-primary bootstrap-invite-button" @label="bootstrap_invite_button_title" /></div>
```

Comment:
```suggestion
      <div class="alert--button"><DButton @icon="user-plus" @action={{this.inviteUsers}} @class="btn-primary bootstrap-invite-button" @label="bootstrap_invite_button_title" /></div>
```

It seems like passing a string in is deprecated and a warning is logged in the console if we do this.

Reply:
Thanks for the suggestion, made the change!